gamification/process/update.py

```diff
@@ -135,6 +136,33 @@ def get_participant_objects():
     return participants
 
 
+def get_meta_review_participant_objects():
+    return meta_review_participants.objects.all()
+
+
+def update_participants_data_with_meta_review(participant):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    received or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
```

Comment:
First check if this participant is in the [active_newcomer_list](https://github.com/coala/community/blob/da5ada362c8fc3ae2cb179739a4babbfad1e7d1f/data/newcomers.py#L9), as gamification only work for those participant.
```
if participant.login in active_newcomers:
     ....
```

Reply:
Thanks!

Reply:
While adding the tests, I'm receiving this traceback 
``` 
    def update_participants_data_with_meta_review(meta_review_participant):
        """
        Update participants based on meta-review

        This method updates every participant based on the meta-review
        received or given. If the meta-review is completed then it calls
        'Did a meta-review or received a meta-review' activity.
        """
>       if meta_review_participant.login in active_newcomers:
E       AttributeError: 'Participant' object has no attribute 'login'

gamification/process/update.py:152: AttributeError
```

@sks444

Reply:
@shikharvaish28 It's hard to tell the reason unless you provide all code, but I guess it's because the `meta_review_participant` you are using here is NOT a meta-review participant, but a gamification participant, which does not have attribute `login`.

Reply:
@li-boxuan @sks444 Please have a look!